Add notes for manual override allowed (#427)
seamapi/docs: docs/products/thermostats/README.md

### Create Thermostat Schedules

> Schedule the climate presets that you have created. Simply specify the key of the climate preset, along with a starting and ending timestamp. You can also indicate whether a person at the thermostat or using the API can change the thermostat's settings after the activation of the climate preset. Further, you can set the maximum override period in minutes. At the end of this override period, Seam sets the thermostat back to the active climate preset.

Reviewer: ```suggestion
Schedule the climate presets that you have created. Simply specify the key of the climate preset, along with a starting and ending timestamp. You can also indicate whether a person at the thermostat or using the API can change the thermostat's for the during the thermostat schedule. Further, you can set the maximum override period in minutes. At the end of this override period, Seam sets the thermostat back to the climate preset defined by the thermostat schedule.
```

> Schedule the climate presets that you have created. Simply specify the key of the climate preset, along with a starting and ending timestamp. You can also indicate whether a person at the thermostat or using the API can change the thermostat's for the during the thermostat schedule. Further, you can set the maximum override period in minutes. At the end of this override period, Seam sets the thermostat back to the climate preset defined by the thermostat schedule.

Author: ```suggestion
Schedule the climate presets that you have created. Simply specify the key of the climate preset, along with a starting and ending timestamp. You can also indicate whether a person at the thermostat or using the API can change the thermostat's settings for the duration of the thermostat schedule. Further, you can set the maximum override period in minutes. At the end of this override period, Seam sets the thermostat back to the climate preset defined by the thermostat schedule.
```

> Schedule the climate presets that you have created. Simply specify the key of the climate preset, along with a starting and ending timestamp. You can also indicate whether a person at the thermostat or issuing an imperative HVAC or fan mode setting request can change the thermostat's settings after the activation of the climate preset, and you can set the maximum override period in minutes. At the end of this override period, Seam sets the thermostat back to the active climate preset.

Reviewer: Something like this, grammar seems off otherwise

```suggestion
Schedule the climate presets that you have created. Simply specify the key of the climate preset, along with a starting and ending timestamp. You can also indicate whether external changes to the HVAC or fan mode settings, either from the the Seam API or the user of the device can change the thermostat's settings after the activation of the climate preset, and you can set the maximum override period in minutes. At the end of this override period, Seam sets the thermostat back to the active climate preset.
```

Author: ```suggestion
Schedule the climate presets that you have created. Simply specify the key of the climate preset, along with a starting and ending timestamp. You can also indicate whether a person at the thermostat or using the API can change the thermostat's settings after the activation of the climate preset. Further, you can set the maximum override period in minutes. At the end of this override period, Seam sets the thermostat back to the active climate preset.
```
